apps/dataset-ingestion/test.sh

```diff
@@ -1,91 +1,22 @@
 #!/bin/bash
-set -e
 
-bash ../build.sh
-export WORKFLOW_NAME="dataset-ingestion"
-RUNNER_NAME="$(whoami)"
-PREFIX="${WORKFLOW_NAME}_${RUNNER_NAME}"
+set -x
+set -euo pipefail
 
-COCO_NW_NAME="${PREFIX}_coco"
-CELEBA_NW_NAME="${PREFIX}_celeba"
-COCO_DB_NAME="${PREFIX}_aperturedb_coco"
-CELEBA_DB_NAME="${PREFIX}_aperturedb_celeba"
+source ../../.commonrc
```

Comment:
This will only work if you cd into the app directory. The previous script could be run from anywhere.